quarkchain/tests/test_core.py

```diff
@@ -108,3 +119,25 @@ def test_optional(self):
         b = v.serialize()
         v1 = Uint32Optional.deserialize(b)
         self.assertEqual(v.value, v1.value)
+
+
+class ABigUint(Serializable):
+    FIELDS = [("value", biguint)]
+
+    def __init__(self, value: int):
+        self.value = value
+
+
+class TestBigUint(unittest.TestCase):
```

Comment:
test boundary values like 2 ** (255 * 8) - 1 and 2 ** (255 * 8) - 1  and 0?

Reply:
done